src/devices/Ahtxx/samples/Ahtxx.Sample.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.Threading;
+
+namespace Iot.Device.Ahtxx.Samples
+{
+    /// <summary>
+    /// Samples for Ahtxx
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main entry point
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            const int I2cBus = 1;
+            I2cConnectionSettings i2cSettings = new I2cConnectionSettings(I2cBus, Aht20.DeviceAddress);
+            I2cDevice i2cDevice = I2cDevice.Create(i2cSettings);
+            Aht20 aht20Sensor = new Aht20(i2cDevice);
+
+            while (true)
+            {
+                Console.WriteLine($"{DateTime.Now.ToLongTimeString()}: {aht20Sensor.GetTemperature()}, {aht20Sensor.GetHumidity()}");
```

Comment:
As GetTemperature() returns a Temperature, you should do something like:
```csharp
aht20Sensor.GetTemperature().DegreeCelsius
```
similar for GetHumidity() with .Ratio so you'll display in the unit you want.

Reply:
For the sample I actually prefer using `GetTemperature()` instead of` GetTemperature().DegreesCelsius.` If using just `GetTemperature()` it makes use of the ToString()-method which gives me a nice formatting with a reasonable number of digits and a unit. Same holds for the humidity.
I think that a user would consider the UnitNet API when using the binding and get the desired unit and full double resolution, e.g. by using `.DegreesCelsius`.
Can you agree to that?

Reply:
It's always better to be explicit on the unit when displaying it.
You can't assume it's been created in a specific measure.  And you show the user how to use the actual units. Otherwise, it's a bit magic :-)

Reply:
I like magic... ;) However, I'll change it.
What do you think about this: aht20Sensor.GetTemperature().ToUnit(TemperatureUnit.DegreeCelsius).ToString() 
I think this meets you needs. Okay? If so, I would do commit/push soon.

Reply:
I did it, as you suggest as this is slightly shorter.
I'll push the change.